lib/pbench/server/api/__init__.py

```diff
@@ -133,6 +134,13 @@ def register_endpoints(api: Api, app: Flask, config: PbenchServerConfig):
         endpoint="key",
         resource_class_args=(config,),
     )
+    api.add_resource(
+        QuisbyData,
+        f"{base_uri}/quisby/<string:dataset>",
+        f"{base_uri}/quisby/<string:dataset>/",
```

Comment:
Without a second URI parameter to decode, I don't think we need or want the trailing `/` form here.

And continuing the "what do we call it?" ... we're presumably going to have a second Quisby-based API to generate comparison data, and we don't want to end up with "quisby-a" and "quisby-b" or unnaturally overloading a single end point.

I think `GET /api/v1/visualize/d1` and `GET /api/v1/compare?datasets=d1,d2,d3` would probably work.